pom.xml

```
@@ -58,7 +58,15 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
-		</plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>14</source>
```

Nie miała być 15?
Dobrym sposobem jest posiadanie jednej zmiennej i wykorzystanie jej w 3ch miejscach:
source, target i java.version dla spring-a.
W ogóle ten kawałek kodu jest niepotrzebny. Wystarczy sama deklaracja odpowiednio nazwanych zmiennych:

```suggestion
                    <properties>
                        <java.version>15</java.version>
                        <maven.compiler.target>${java.version}</maven.compiler.target>
	                <maven.compiler.source>${java.version}</maven.compiler.source>
                    </properties>
```

Tylko ten snippet zamiast całego <plugin> ;)